packs/github/actions/create_issue.py

```diff
@@ -0,0 +1,18 @@
+from lib.base import BaseGithubAction
+from github import GithubObject
+from lib.formatters import issue_to_dict
+
+__all__ = [
+    'CreateIssueAction'
+]
+
+
+class CreateIssueAction(BaseGithubAction):
+    def run(self, user, repo, title, description, assignee):
```

Comment:
Have you seen my or statements that take care of that?
Or do I mixup something?